ft-sdk/src/auth/schema.rs

```diff
@@ -20,6 +20,7 @@ diesel::table! {
         data -> Text,
         updated_at -> Timestamptz,
         created_at -> Timestamptz,
+        expires_at -> Nullable<Timestamptz>,
```

Comment:
Cookie will surely delete it but how are we going to delete this from db?

Reply:
A background worker.

Reply:
So a background worker should read from db when the session should expire and then delete it. If my understanding is correct then we need `expires_at` column in the `fastn_session` table.

Reply:
As discussed, yes. But since there are multiple possible behaviours for session expiry, let's not tackle session expiry related stuff now. First let's have a discussion and full design on session expiry then we write code.